### PR TITLE
OADP-3486: Fix garbage collection frequency description

### PR DESCRIPTION
**Pull request by PrasadJoshi12**

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

File: bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml [hunk 1]
```diff
@@ -546,7 +546,7 @@ spec:
                               format: int64
                               type: integer
                             garbage-collection-frequency:
-                              description: How long to wait by default before backups can be garbage collected. (default 720h0m0s)
+                              description: How long to wait by default before backups can be garbage collected. (gc-controller runs its reconciliation loop every hour by default)
```
Comment (sseago) [inline bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml:549]: This fixes the duration problem, but the description text is wrong. This isn't how long to wait but how frequently to run, and it has nothing to do with defaults. Best to replace "How long to wait by default before backups can be garbage collected" with "How frequently Velero checks for expired backups to delete." or something similar

**Review by sseago — CHANGES_REQUESTED** [review]

**Comment by PrasadJoshi12** [comment]
Thank you @kaovilai @weshayutin @sseago !

File: bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml (hunk 1 above)
Comment (PrasadJoshi12) [reply bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml:549]: sure

Comment (PrasadJoshi12) [reply bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml]: @sseago Can you please take a look ?

File: bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml [hunk 2]
```diff
@@ -546,7 +546,7 @@ spec:
                               format: int64
                               type: integer
                             garbage-collection-frequency:
-                              description: How long to wait by default before backups can be garbage collected. (default 720h0m0s)
+                              description: How often garbage collection is run for expired backups. (default once per hour)
```
Comment (kaovilai) [inline bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml:549]: More clarity like so? almost :)
```suggestion
                              description: How often garbage collection checks for expired backups. (default once per hour)
```

Comment (PrasadJoshi12) [reply bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml:549]: Thanks @kaovilai, I will patch it

**Review by kaovilai — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Comment by mateusoliveira43** [comment]
Please rebase, so changes from https://github.com/openshift/oadp-operator/commit/314cb261c59ac3b5e6636d579d4bec490accff75 are applied first

**Comment by PrasadJoshi12** [comment]
@mateusoliveira43 can you please review ?

File: pkg/velero/server/config.go [hunk 3]
```diff
@@ -75,7 +75,7 @@ type ServerConfig struct {
 	// How often (in nanoseconds) 'maintain' is run for backup repositories by default.
 	// +optional
 	RepoMaintenanceFrequency *time.Duration `json:"default-repo-maintain-frequency,omitempty"`
-	// How long (in nanoseconds) to wait by default before backups can be garbage collected. (default is 720 hours)
+	// How often (in nanoseconds) garbage collection checks for expired backups. (default once per hour)
```
Comment (mateusoliveira43) [inline pkg/velero/server/config.go:78]: I would change "default once per hour" to "default is 1 hour" for consistency. I believe all other places uses this syntax

**Review by mateusoliveira43 — APPROVED** [review]

File: pkg/velero/server/config.go (hunk 3 above)
Comment (PrasadJoshi12) [reply pkg/velero/server/config.go:78]: Sure, I will patch it.

Comment (PrasadJoshi12) [reply pkg/velero/server/config.go]: Done.

**Review by mateusoliveira43 — APPROVED** [review]
/lgtm

**Review by kaovilai — APPROVED** [review]

**Pull request merged**